content/notero-item.ts

```diff
@@ -126,6 +126,10 @@ export default class NoteroItem {
     return Number.isNaN(year) ? null : year;
   }
 
+  public getDate(): string {
+    return this.zoteroItem.getField('date') || '';
```

Comment:
For consistency with the other fields, could we change this to return `null` instead of empty string when no date is present? As far as I can tell, it should work the same either way, but I think I'd prefer to keep the return type consistent with other fields like this so I don't come back to this code months down the road and try to remember if there was a functional reason for making this one different. 😄 

```suggestion
  public getDate(): string | null {
    return this.zoteroItem.getField('date') || null;
```